dali/operators/bbox/bbox_paste.cc

```diff
@@ -35,22 +36,23 @@ Where::
   paste_x' = paste_x * (ratio - 1)/ratio
   paste_y' = paste_y * (ratio - 1)/ratio
 
-Paste coordinates are normalized so that `(0,0)` aligns the image to top-left of the canvas and `(1,1)` aligns it to bottom-right.
+The paste coordinates are normalized so that ``(0,0)`` aligns the image to top-left of the
+canvas and ``(1,1)`` aligns it to bottom-right.
 )code")
   .NumInput(1)
   .NumOutput(1)
   .AddArg("ratio",
-      R"code(Ratio of canvas size to input size, must be > 1.)code",
+      R"code(RRatio of the canvas size to the input size, and the value must be greater than 1.)code",
```

Comment:
```suggestion
      R"code(Ratio of the canvas size to the input size; the value must be at least 1.)code",
```
If ratio is 1, the paste is a no-op, but the value should still be legal. It's never validated in the code, BTW.

Reply:
Done